usr/local/www/diag_dump_states.php

```diff
@@ -139,42 +132,36 @@ function(index,row) { jQuery(row).fadeOut(1000); }
 	$current_statecount=`pfctl -si | grep "current entries" | awk '{ print $3 }'`;
 ?>
 
-<table class="tabcont" width="100%" border="0" cellspacing="0" cellpadding="0" summary="states">
-	<tr>
-		<td>
-			<form action="<?=$_SERVER['SCRIPT_NAME'];?>" method="post" name="iform">
-			<table class="tabcont" width="100%" border="0" cellspacing="0" cellpadding="0" summary="filter">
-				<tr>
-					<td>
-						<?=gettext("Current total state count");?>: <?= $current_statecount ?>
-					</td>
-					<td style="font-weight:bold;" align="right">
-						<?=gettext("Filter expression:");?>
-						<input type="text" name="filter" class="formfld search" value="<?=htmlspecialchars($_POST['filter']);?>" size="30" />
-						<input type="submit" class="formbtn" value="<?=gettext("Filter");?>" />
-					<?php if (isset($_POST['filter']) && (is_ipaddr($_POST['filter']) || is_subnet($_POST['filter']))): ?>
-						<input type="submit" class="formbtn" name="killfilter" value="<?=gettext("Kill");?>" />
-					<?php endif; ?>
-					</td>
-				</tr>
-			</table>
-			</form>
-		</td>
-	</tr>
-	<tr>
-		<td>
-			<table class="tabcont sortable" width="100%" border="0" cellspacing="0" cellpadding="0" summary="results">
-				<thead>
-				<tr>
-					<th class="listhdrr" width="5%"><?=gettext("Int");?></th>
-					<th class="listhdrr" width="5%"><?=gettext("Proto");?></th>
-					<th class="listhdrr" width="65"><?=gettext("Source -> Router -> Destination");?></th>
-					<th class="listhdr" width="24%"><?=gettext("State");?></th>
-					<th class="list sort_ignore" width="1%"></th>
-				</tr>
-				</thead>
-				<tbody>
+	<form action="<?=$_SERVER['SCRIPT_NAME']?>" method="post" name="iform">
+	<?=gettext("&nbsp;Current total state count")?>: <?= $current_statecount ?>
+	<div class="pull-right">
+		<?=gettext("Filter expression:")?>
+		<input type="text" name="filter" class="formfld search" value="<?=htmlspecialchars($_POST['filter'])?>" size="30" />
+		<input type="submit" class="btn btn-primary btn-sm formbtn" value="<?=gettext("Filter")?>" />
+
+<?php
+		if (isset($_POST['filter']) && (is_ipaddr($_POST['filter']) || is_subnet($_POST['filter'])))
+		   print('<input type="submit" class="btn btn-danger btn-sm formbtn" name="killfilter" value="Kill">');
+?>
+
+	&nbsp;
+	</div>
+
+	</form>
+	<br>
```

Comment:
Please remove this and other `<br>`'s which are only used for spacing. That's _not_ what this element is for.